src/DataTypes/string.tsx

```diff
@@ -0,0 +1,115 @@
+import { CreateFilter, getDataTypeSchema, regexEscape } from './utils';
+import { FULL_TEXT_SLUG } from '../components/Filters/SchemaSieve';
+import { JSONSchema7 as JSONSchema } from 'json-schema';
+
+// TODO: since we have the rendererSchema we can probably make it a normal object
+export const operators = () => ({
```

Comment:
I was wondering about making this a normal object too

Reply:
let me change the todo, unfortunately, until we will have a custom logic for Tags in the DataTypes/object, it will be better to keep every operators in the same format so that we can use the same signature without having to add checks